src/main/kotlin/value/VObj.kt

```diff
@@ -51,4 +54,24 @@ data class VObj(val v: Obj.ID?): Value() {
 
     private fun propTraits(obj: Obj) = VList.make(obj.traits.mapNotNull { it.trait()?.vTrait })
 
+    override fun callStaticVerb(c: Context, name: String, args: List<Value>) = when (name) {
+        "hasProp" -> verbHasProp(args)
+        "clearProp" -> verbClearProp(args)
+        else -> null
+    }
+
+    private fun verbHasProp(args: List<Value>): VBool {
```

Comment:
Dumb question: why is the value passed in a list?

Reply:
verbs called from yeggcode could have been called with any number of args (you could write "myobj.hasProp("x", "duh")" by mistake or some such).  verbHasProp here is what handles that call, so it has to take the whole list of args the user wrote in the yeggcode.